index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <div class="body-container">
+      <div class="toolbar">
+        <a href="#" class="logo">John Doe</a>
+      <nav class="navbar-container">
+        <ul>
+          <li><a class = "hamburger" href="javascript:void(0);"><i class="fa fa-bars"></i></a></li>
+        </ul>
+      </nav>
+      </div>
+      <div>
+        <img src="images/ellipse18.png" class="ellipse-image-one" alt="ellipse-image-one"/>
+      </div>
+      <div class="section1-details">
+        <section>
+            <h2>Paul Onchera</h2>
```

Comment:
- [ ] [RQUIRED] I think it would be better if you use an `<h1>` element instead of the `<h2>` element ♻️ to emphasize that this is the most important heading in the page 🌟.